sdk/storage/Azure.Storage.Blobs/src/autorest.md

```diff
@@ -324,10 +324,19 @@ directive:
 ```
 
 ### Don't buffer downloads and query
-
 ``` yaml
 directive:
 - from: swagger-document
   where: $..[?(@.operationId=='Blob_Query' || @.operationId=='Blob_Download')]
   transform: $["x-csharp-buffer-response"] = false;
+```
+
+### Fix XML string "ObjectReplicationMetadata" to "OrMetadata"
```

Comment:
This will be removed when the autorest version gets updated (https://github.com/Azure/azure-sdk-for-net/pull/24379). Updating the autorest version removes the need for this.

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/24485